src/SessionHandler/Psr6SessionHandler.php

```diff
@@ -36,41 +37,63 @@ class Psr6SessionHandler implements \SessionHandlerInterface
     /**
      * @param CacheItemPoolInterface $cache
      * @param array                  $options {
+     * @type int $ttl The time to live in seconds
+     * @type string $prefix The prefix to use for the cache keys in order to avoid collision
+     *                                        }
      *
-     *      @type int $ttl       The time to live in seconds
-     *      @type string $prefix The prefix to use for the cache keys in order to avoid collision
-     * }
+     * @throws \InvalidArgumentException
      */
     public function __construct(CacheItemPoolInterface $cache, array $options = [])
     {
         $this->cache = $cache;
 
+        if ($diff = array_diff(array_keys($options), ['prefix', 'ttl'])) {
+            throw new \InvalidArgumentException(sprintf(
+                'The following options are not supported "%s"', implode(', ', $diff)
+            ));
+        }
+
         $this->ttl    = isset($options['ttl']) ? (int) $options['ttl'] : 86400;
         $this->prefix = isset($options['prefix']) ? $options['prefix'] : 'psr6ses_';
     }
 
     /**
      * {@inheritdoc}
      */
-    public function open($savePath, $sessionName)
+    public function close()
     {
         return true;
     }
 
     /**
      * {@inheritdoc}
      */
-    public function close()
+    public function gc($lifetime)
```

Comment:
Shouldnt this part of the AbstractSessionHandler?

Reply:
i just forgot it